### PR TITLE
Getting Started

### PR DESCRIPTION
**Pull request by dzucconi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,17 @@ The general idea here is something minimal, utilitarian, unobtrusive and adaptib
 * * *
 
 ### Local installation
+
+```bash
 	git clone git@github.com:arenahq/ervell.git
 	cd ervell
 	npm install
```
Comment (dzucconi) [inline README.md]: D'oh actual tabs

File: README.md [hunk 2]
```diff
@@ -7,9 +7,17 @@ The general idea here is something minimal, utilitarian, unobtrusive and adaptib
...
+  gem install foreman
+```
 
 ### Running the server
-	make s
\ No newline at end of file
+
+```bash
+  foreman start -f Procfile.dev -e .env.dev
```
Comment (broskoski) [inline README.md]: It's a new day

Comment (dzucconi) [inline README.md]: This is just ergonomics for me. `make spc` doesn't start up Redis.

Comment (broskoski) [inline README.md]: I'm down!

**Pull request merged**